bundles/org.eclipse.osgi/osgi/src/org/osgi/framework/connect/ConnectFramework.java

```diff
@@ -0,0 +1,25 @@
+package org.osgi.framework.connect;
```

Comment:
What do you suggest? Can we add a new interface or will this also be complained? this might be placed in an equinox specific package then and one need to use instanceof so we can keep the original signatures here?